logos-cli/tests/data/no_error_lut/fmt_output.rs

```diff
@@ -0,0 +1,195 @@
+                Some([128u8..=159u8, 128u8..=191u8]) => {
+                    lex.bump_unchecked(2usize);
+                    goto2_ctx11_x(lex)
+                }
+                _ => goto11_ctx11_x(lex),
+            }
+        }
+        #[inline]
+        fn goto17_ctx11_x<'s>(lex: &mut Lexer<'s>) {
+            match lex.read::<&[u8; 3usize]>() {
+                Some([144u8..=191u8, 128u8..=191u8, 128u8..=191u8]) => {
+                    lex.bump_unchecked(3usize);
+                    goto2_ctx11_x(lex)
+                }
+                _ => goto11_ctx11_x(lex),
+            }
+        }
+        #[inline]
+        fn goto2_x<'s>(lex: &mut Lexer<'s>) {
+            lex.set(Ok(Token::AnyUnicode));
+        }
+        #[inline]
+        fn goto13_ctx11_x<'s>(lex: &mut Lexer<'s>) {
+            match lex.read::<&[u8; 1usize]>() {
+                Some([128u8..=191u8]) => {
+                    lex.bump_unchecked(1usize);
+                    goto2_ctx11_x(lex)
+                }
+                _ => goto11_ctx11_x(lex),
+            }
+        }
+        #[inline]
+        fn goto18_ctx11_x<'s>(lex: &mut Lexer<'s>) {
+            match lex.read::<&[u8; 3usize]>() {
+                Some([128u8..=191u8, 128u8..=191u8, 128u8..=191u8]) => {
+                    lex.bump_unchecked(3usize);
+                    goto2_ctx11_x(lex)
+                }
+                _ => goto11_ctx11_x(lex),
+            }
+        }
+        #[inline]
+        fn goto15_ctx11_x<'s>(lex: &mut Lexer<'s>) {
+            match lex.read::<&[u8; 2usize]>() {
+                Some([128u8..=191u8, 128u8..=191u8]) => {
+                    lex.bump_unchecked(2usize);
+                    goto2_ctx11_x(lex)
+                }
+                _ => goto11_ctx11_x(lex),
+            }
+        }
+        #[inline]
+        fn goto14_ctx11_x<'s>(lex: &mut Lexer<'s>) {
+            match lex.read::<&[u8; 2usize]>() {
+                Some([160u8..=191u8, 128u8..=191u8]) => {
+                    lex.bump_unchecked(2usize);
+                    goto2_ctx11_x(lex)
+                }
+                _ => goto11_ctx11_x(lex),
+            }
+        }
+        #[inline]
+        fn goto19_ctx11_x<'s>(lex: &mut Lexer<'s>) {
+            match lex.read::<&[u8; 3usize]>() {
+                Some([128u8..=143u8, 128u8..=191u8, 128u8..=191u8]) => {
+                    lex.bump_unchecked(3usize);
+                    goto2_ctx11_x(lex)
+                }
+                _ => goto11_ctx11_x(lex),
+            }
+        }
+        #[inline]
+        fn goto11_x<'s>(lex: &mut Lexer<'s>) {
+            lex.set(Ok(Token::Any));
+        }
+        #[inline]
+        fn goto20<'s>(lex: &mut Lexer<'s>) {
+            enum Jump {
+                J1,
+                J16,
+                J17,
+                J2,
+                J13,
+                J18,
+                J15,
+                J14,
+                J19,
+                J11,
+            }
```

Comment:
Note how there is no `__` variant here, which means that the error variant is omitted.